### ISSUE TITLE
Power monitor reporting 20RF/tick less than it should

### ISSUE THREAD

**Issue by BillH1949** [SleepyTrousers/EnderIO#1743]
In my network I have 48 combustion engines powered by rocket fuel and connected by Ender energy conduits to a 1x2 Vibrant capacitor bank and a 4x4 Basic capacitor bank.  My power monitor provides redstone control for these engines.  When they start up, the input initally displays 7680 RF/tick as expected, but over a few seconds drops to 7660 RF/tick.  I've disconnected all but two engines from the network to see if it was a size issue but the power loss remains consistent.

**Comment by 6210classick** [comment]
doesn't the power monitor consume RF to operate ?

**Comment by BillH1949** [comment]
Of course, but I'm fairly certain I got an accurate reading earlier in the game when I used enhanced energy conduit.  Much as I'm loathe to mess with my setup again, I'll try switching that out and see what happens.

**Comment by BillH1949** [comment]
Hm. I just switched out the conduits and I still get a 20RF/tick difference.

**Comment by BillH1949** [comment]
OK. I made another power monitor and tried a simple test using a Stirling Generator (with octadic capacitor) and a Slice'n'Splice, all with empty internal buffers.
![power monitor test](https://cloud.githubusercontent.com/assets/10668463/5883528/2ae9c8dc-a308-11e4-8ac5-7f4afa033a7b.png)

At first, the power monitor read 40RF/tick in and 40RF/tick out.  The power monitor's buffer filled first and then the reading changed to 
![power monitor display](https://cloud.githubusercontent.com/assets/10668463/5883583/a4d0e036-a308-11e4-829b-9f89a9b8360a.png)

Once the Slice'n'Splice's buffer filled, the input and output dropped to 0 as expected.

**Comment by MatthiasMann** [comment]
The power monitor will only monitor energy flow in a conduit. But when you place machines next to a generator / capacitor bank then these will also output energy directly to these machines. That directly transmitted energy will not be visible to the power monitor.

**Comment by BillH1949** [comment]
I understand that.  My engines and machines are not adjacent to my capacitor bank, they are only connected via conduits.  I do have the power monitor and a dimensional transceiver adjacent, but the capacitor sides are disabled next to those elements.  In any case, I've tried moving the power monitor away from the bank and removing the transceiver and I still get the 20RF difference, so that is not the issue.  Given that the engines (the power source) are connected via conduits, I expect that the input energy should be correctly displayed.

**Comment by BillH1949** [comment]
More testing with this.  With the basic setup shown in the screenshot below, the power monitor displays correctly:
![test setup 1](https://cloud.githubusercontent.com/assets/10668463/6102013/53df7060-afe3-11e4-86e9-fa1e4d10e64b.png)

However, at my base with 64 combustion engines, I'm still seeing a 20RF/tick discrepancy:
![power monitor display](https://cloud.githubusercontent.com/assets/10668463/6102016/785ce332-afe3-11e4-802e-9c38d55f3a12.png)

**Comment by BillH1949** [comment]
Did some more testing.  I added one of each type of power-using machine I use at my base (2 SAG mills, actually, alloy smelter, painter, crafter, soul binder, attraction obelisk, slice'n'splice, farming station) to see if one of them had a hidden drain; nope.  I then incrementally added combustion engines until I got to four.  As I added them, I started noticing a slight discrepancy in the power monitor:
![ss 1](https://cloud.githubusercontent.com/assets/10668463/6122589/afb7d1b2-b0aa-11e4-825e-7b9e7067487b.png)
![ss 2](https://cloud.githubusercontent.com/assets/10668463/6122593/b50eb522-b0aa-11e4-9e8f-a02ab539a7ca.png)

I cycled the power (I had the engines under redstone control with a switch) and then I got the display I was expecting (full input from the engines, 1 RF/tick drain from the power monitor):
![ss 3](https://cloud.githubusercontent.com/assets/10668463/6122603/e39bb8fe-b0aa-11e4-9df2-f1067de55999.png)

One thing I noticed here is that the in/out discrepancy occurred while the machine buffers were filling, and the capacitor didn't start filling until the machine buffers were full.  At my base, it seems the machine buffers don't ever completely fill for some reason, though the capacitor is being charged.

Hope this analysis is helping.
